src/components/typography/index.ts

```diff
@@ -0,0 +1,4 @@
+export { default } from "./typography.component";
+export { List, ListItem } from "./list.component";
+export type { TypographyProps } from "./typography.component";
+export type { ListProps } from "./list.component";
```

Comment:
```suggestion
export type { ListProps, ListItemProps } from "./list.component";
```

See my comment about ListItem having slightly different prop interface than List